src/components/KSelect/KSelect.vue

```diff
@@ -271,18 +280,35 @@ export default defineComponent({
     const selectInputId = computed((): string => props.testMode ? 'test-select-input-id-1234' : uuidv1())
     const selectTextId = computed((): string => props.testMode ? 'test-select-text-id-1234' : uuidv1())
     const selectItems: Ref<SelectItem[]> = ref([])
+    const filterIsEnabled = computed((): boolean => {
+      if (props.enableFiltering !== null) {
```

Comment:
this would need to be updated once the default prop value is changed to a Boolean

Reply:
See above